eox_tenant/constants.py

```diff
@@ -5,3 +5,9 @@
 CMS_ENVIRONMENT = getattr(settings, "SERVICE_VARIANT", None) == "cms"
 LMS_CONFIG_COLUMN = "lms_configs"
 CMS_CONFIG_COLUMN = "studio_configs"
+BASE_ALLOWED_APPLICATIONS = [
+    "cms-sso",
+    "cms-sso-dev",
+    "superset-sso",
```

Comment:
I'm fine with CMS as the base allowed application, but I don't think aspects should be in the defaults. Many instances don't use aspects, so I think it would be better to keep CMS as the default and make instances explicitly add the setting when they need aspects

Reply:
My reasoning is that CMS/Studio is present in all instances. But Aspects is a separate and optional service, not part of the core platform, and not used by many instances. I think it's cleaner to only configure settings for services that are actually present, rather than having default configurations for optional services that may not be installed